src/client/channelOwner.ts

```diff
@@ -78,7 +78,7 @@ export abstract class ChannelOwner<T extends channels.Channel = channels.Channel
         if (prop === 'debugScopeState')
           return (params: any) => this._connection.sendMessageToServer(this, prop, params, stackTrace);
         if (typeof prop === 'string') {
-          const validator = scheme[paramsName(this._type, prop)];
+          const validator = scheme[paramsName(this._type === '' ? 'Root' : this._type, prop)];
```

Comment:
Why don't we change Root constructor to pass 'Root' type instead?

Reply:
yeah was a hacky workaround, calling it actually correctly is way better.